Sources/PackageModelSyntax/CMakeLists.txt

```diff
@@ -24,6 +24,7 @@ target_link_libraries(PackageModelSyntax PUBLIC
   Basics
   PackageLoading
   PackageModel
+  ProductDescription
```

Comment:
I'm confused by this. There is no library called ProductDescription. Did you mean PackageDescription here?

Reply:
Yeah not sure what I was thinking here, I mixed these up. Just pushed change to `PackageDescription`